### PR TITLE
Adding BRACKEN module and functionality

### PR DESCRIPTION
**Pull request by kylacochrane**
This PR is meant to add BRACKEN functionality to the SpeciesAbundance pipeline.

A new mini database has been generated from two bacterial genomes = 18MB.
The testing input samplesheet has been reverted to the nf-core/test_datasets fastq files from the `mag` pipeline. 

Some files (not directly related to the BRACKEN module) have had formatting updates for consistency.

### CODE REVIEW

**Comment by kylacochrane** [comment]
I just realized I added the adjust_bracken_for_unclassified_reads.py script to `bin/` in this PR by mistake. It was meant for another PR for the next module in the pipeline.

File: modules/local/bracken/main.nf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    bracken \\
+        ${args} \\
+        -d ${bracken_db} \\
+        -t $task.cpus \\
+        -i ${report_txt} \\
+        -w ${meta.id}_${taxonomic_level}_bracken.txt \\
+        -o ${meta.id}_${taxonomic_level}_bracken_abundances_unsorted.tsv \\
+        -l ${taxonomic_level}
+
+    paste <(echo "meta.id") <(head -n 1 ${meta.id}_${taxonomic_level}_bracken_abundances_unsorted.tsv) | tr \$'\\t' ',' > bracken_abundances_header.csv
```
Comment (apetkau) [inline modules/local/bracken/main.nf:38]: Could you describe what this command is doing?

**Review by apetkau — CHANGES_REQUESTED** [review]
This is amazing @kylacochrane . Thanks so much for this. And thank you so much for building the kraken2 database with the files for bracken as well 😄 .

I have one comment below.

Additionally, when it's time to merge, one suggestion I have is to **Squash and merge** instead of merging: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request#merging-a-pull-request. This will combine all the commits into this branch into a single commit, which means that the previously uploaded large fastq files won't exist in the history (and so the git repo will be smaller).

This has the downside, though, of removing all the intermediate commits in this branch. Let me know what you think.

**Review by apetkau — APPROVED** [review]

File: modules/local/bracken/main.nf (hunk 1 above)
Comment (kylacochrane) [reply modules/local/bracken/main.nf:39]: Added comment to describe: [6c15dc9](https://github.com/phac-nml/speciesabundance/pull/7/commits/6c15dc9f5d581bd6899a46f94fcd1bd8dffaf2e7)

Comment (apetkau) [reply modules/local/bracken/main.nf:39]: Thanks so much 😄

**Pull request merged**